FundParser/BL/Services/HoldingDiffCalculatorService/HoldingDiffCalculatorService.cs

```diff
@@ -24,7 +24,7 @@ public class HoldingDiffCalculatorService : IHoldingDiffCalculatorService
             {
                 var newHolding = newHoldingDict.GetValueOrDefault(h.CompanyId);
 
-                return newHolding == null ? GetOldHoldingDiff(h) : GetHoldingDiff(h, newHolding);
+                return newHolding is null ? GetOldHoldingDiff(h) : GetHoldingDiff(h, newHolding);
```

Comment:
We have some null checks that are using the basic `==` or `!=`. We should either change the ones using `is null` or change the others to be in sync. Though I guess we don't really have to deal with it.